### PR TITLE
Added errorMessage partial to user sessions form

### LINKED ISSUE #8959: Add errorMessage partial in user_sessions/_form.html.erb
This issue is a part of #4622. The errorMessage partial has been merged in PR #8923

Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [app/views/user_sessions/_form.html.erb]($BRANCH_URL) in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
<center>
<h2><%= raw translation('user_sessions.new.log_in') %></h2>
  <p class="form-grey" id="toSignup">or
    <%= link_to signup_path( return_to: params[:return_to] ) do %>
    <%= raw translation('user_sessions.new.sign_up') %></a> <%= raw translation('user_sessions.new.to_join') %>
    <% end %></p>
</center>

- <% if f.error_messages != "" %><div class="alert alert-danger"><%= f.error_messages %></div><% end %>
+ <%= render partial: 'layouts/errorMessages', locals: { model: @user_session } %>

<br style="clear:both;" />
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by gucci-ninja**
Fixes #8959 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by noi5e** [comment]
Hi @gucci-ninja! I took a look at the tests that were failing... Hmm... That's weird, they don't seem to be related to what you're working on.

Just FYI (and maybe you know this already), you can look at the test results here by clicking on Details:
Screen Shot 2021-01-07 at 6 22 15 PM
Then the stacktrace will give you the place to look in the codebase to do some more research (line 115 in this file):
Screen Shot 2021-01-07 at 6 23 18 PM

Sometimes the tests will fail not necessarily because of the PR. If they fail relatively quickly, like 5-10 minutes, you can run them again by clicking on Re-run jobs here:
Screen Shot 2021-01-07 at 6 26 30 PM

I just tried re-running them for you, let's see what happens!

**Comment by gucci-ninja** [comment]
@noi5e yeah I noticed so I haven't put the PR up for review yet. Lots of undefined method 'errors' for nil:NilClass errors. I also couldn't find a @user_session defined in the controller so that might be causing it.

**Comment by noi5e** [comment]
Hmm, weird. I did notice that the continuous integration tests that were failing the second time were different, and there were new errors that weren't there the first time.

I think it would help reviewing your code if you could paste screenshots of the Rails stacktrace `nil` errors?

File: app/controllers/user_sessions_controller.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 class UserSessionsController < ApplicationController
   before_action :require_no_user, only: [:new]
   def new
+    @user_session = UserSession.new
```
Comment (noi5e) [inline app/controllers/user_sessions_controller.rb:4]: It seems like this wasn't specified in the original issue? Did you take a look at the user sessions controller [here](https://github.com/publiclab/plots2/blob/main/app/controllers/user_sessions_controller.rb)?

Sorry I can't be more helpful, this isn't really my expertise, but perhaps @cesswairimu @jywarren or @sagarpreet-chadha could help.

Comment (gucci-ninja) [reply app/controllers/user_sessions_controller.rb:4]: Yeah I added that in after noticing the tests were failing. Thanks a lot for looking into this btw. I'll post the stack trace as well

**Comment by coder645** [comment]
@gucci-ninja it's not because of `@user_session` not being in the controller. I added that and tried but still gives errors. Ig this is a general problem and is caused because of another PR not being merged.
Could you close this PR for now? I'll ask to reopen the issue after this problem is resolved.
Cheers!

**Comment by gucci-ninja** [comment]
@coder645 Sounds good! I'll keep an eye out for when it reopens! Thanks for your help everyone :100:

**Comment by cesswairimu** [comment]
Hi @gucci-ninja, there are these new ftos #8974 and #8975 if you are still looking for another fto. Thanks

**Pull request closed without merging**